android/src/org/mavlink/qgroundcontrol/QGCUsbId.java

```diff
@@ -0,0 +1,125 @@
+package org.mavlink.qgroundcontrol;
+
+public final class QGCUsbId {
+
+    public static final int VENDOR_FTDI = 0x0403;
+    public static final int FTDI_FT232R = 0x6001;
+    public static final int FTDI_FT2232H = 0x6010;
+    public static final int FTDI_FT4232H = 0x6011;
+    public static final int FTDI_FT232H = 0x6014;
+    public static final int FTDI_FT231X = 0x6015; // same ID for FT230X, FT231X, FT234XD
+
+    public static final int VENDOR_SILABS = 0x10c4;
+    public static final int SILABS_CP2102 = 0xea60; // same ID for CP2101, CP2103, CP2104, CP2109
+    public static final int SILABS_CP2105 = 0xea70;
+    public static final int SILABS_CP2108 = 0xea71;
+
+    public static final int VENDOR_PROLIFIC = 0x067b;
+    public static final int PROLIFIC_PL2303 = 0x2303;   // device type 01, T, HX
+    public static final int PROLIFIC_PL2303GC = 0x23a3; // device type HXN
+    public static final int PROLIFIC_PL2303GB = 0x23b3; // "
+    public static final int PROLIFIC_PL2303GT = 0x23c3; // "
+    public static final int PROLIFIC_PL2303GL = 0x23d3; // "
+    public static final int PROLIFIC_PL2303GE = 0x23e3; // "
+    public static final int PROLIFIC_PL2303GS = 0x23f3; // "
+
+    public static final int VENDOR_GOOGLE = 0x18d1;
+    public static final int GOOGLE_CR50 = 0x5014;
+
+    public static final int VENDOR_QINHENG = 0x1a86;
+    public static final int QINHENG_CH340 = 0x7523;
+    public static final int QINHENG_CH341A = 0x5523;
+
+    public static final int VENDOR_UNISOC = 0x1782;
+    public static final int FIBOCOM_L610 = 0x4D10;
+    public static final int FIBOCOM_L612 = 0x4D12;
+
+    //////////////////////////////////////////////
+
+    public static final int VENDOR_PX4 = 0x26AC;
+    public static final int DEVICE_PX4FMU_V1 = 0x0010;
+    public static final int DEVICE_PX4FMU_V2 = 0x0011;
+    public static final int DEVICE_PX4FMU_V3 = 0x0011;
+    public static final int DEVICE_PX4FMU_V4 = 0x0012;
+    public static final int DEVICE_PX4FMU_V4PRO = 0x0013;
+    public static final int DEVICE_PX4FMU_V5 = 0x0032;
+    public static final int DEVICE_PX4FMU_V5X = 0x0033;
+    public static final int DEVICE_PX4FMU_V6C = 0x0038;
+    public static final int DEVICE_PX4FMU_V6U = 0x0036;
+    public static final int DEVICE_PX4FMU_V6X = 0x0035;
+    public static final int DEVICE_PX4FMU_V6XRT = 0x001D;
+    public static final int DEVICE_PX4MINDPX_V2 = 0x0030;
```

Comment:
It seems like we already carry this information elsewhere, is there any way to avoid duplicating?

Reply:
USBBoardInfo.json, And I could probably come up with a way but I don't know Java well enough to do it easily. Maybe through CMake after we drop QMake.